app/src/main/java/pe/edu/ucsp/vraze/Menu.java

```diff
@@ -0,0 +1,459 @@
+
+  }
+
+  @Override
+  public void onRoomConnected(int statusCode, Room room) {
+    Log.d(TAG, "onRoomConnected(" + statusCode + ", " + room + ")");
+    if (statusCode != GamesStatusCodes.STATUS_OK) {
+      Log.e(TAG, "*** Error: onRoomConnected, status " + statusCode);
+      showGameError();
+      return;
+    }
+    updateRoom(room);
+  }
+
+
+  @Override
+  public void onRoomConnecting(Room room) {
+    updateRoom(room);
+  }
+
+  @Override
+  public void onRoomAutoMatching(Room room) {
+    updateRoom(room);
+
+  }
+    /*fin de todo lo referente a salas de juego*/
+
+
+  @Override
+  public void onPeerInvitedToRoom(Room room, List<String> list) {
+    updateRoom(room);
+
+  }
+
+  // We treat most of the room update callbacks in the same way: we update our list of
+  // participants and update the display. In a real game we would also have to check if that
+  // change requires some action like removing the corresponding player avatar from the screen,
+  // etc.
+  @Override
+  public void onPeerDeclined(Room room, List<String> list) {
+    updateRoom(room);
+
+  }
+
+  @Override
+  public void onPeerJoined(Room room, List<String> list) {
+    updateRoom(room);
+
+  }
+
+  @Override
+  public void onPeerLeft(Room room, List<String> list) {
+    updateRoom(room);
+
+  }
+
+  // Called when we are connected to the room. We're not ready to play yet! (maybe not everybody
+  // is connected yet).
+  @Override
+  public void onConnectedToRoom(Room room) {
+    Log.d(TAG, "onConnectedToRoom.");
+
+    //get participants and my ID:
+    mParticipants = room.getParticipants();
+    mMyId = room.getParticipantId(Games.Players.getCurrentPlayerId(mGoogleApiClient));
+
+    // save room ID if its not initialized in onRoomCreated() so we can leave cleanly before the game starts.
+    if (mRoomId == null)
+      mRoomId = room.getRoomId();
+
+    // print out the list of participants (for debug purposes)
+    Log.d(TAG, "Room ID: " + mRoomId);
+    Log.d(TAG, "My ID " + mMyId);
+    Log.d(TAG, "<< CONNECTED TO ROOM>>");
+  }
+
+  // Called when we get disconnected from the room. We return to the main screen.
+  @Override
+  public void onDisconnectedFromRoom(Room room) {
+    mRoomId = null;
+    showGameError();
+  }
+
+  @Override
+  public void onPeersConnected(Room room, List<String> list) {
+    updateRoom(room);
+  }
+
+  @Override
+  public void onPeersDisconnected(Room room, List<String> list) {
+    updateRoom(room);
+  }
+
+  @Override
+  public void onP2PConnected(String s) {
+//nada
+  }
+
+  @Override
+  public void onP2PDisconnected(String s) {
+//nada
+  }
+
+  // Called when we get an invitation to play a game. We react by showing that to the user.
+
+
+  // Broadcast my score to everybody else.
+  public static void SendToPlayers(float x, float y, float dir0, float dir1, boolean is_finish) {
+    if (!mMultiplayer)
+      return; // playing single-player mode
+
+    mMsgBuf[0] = (byte) x;
```

Comment:
https://stackoverflow.com/questions/14619653/converting-a-float-to-a-byte-array-and-vice-versa-in-java